<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTaskPanelWidget</class>
 <widget class="QWidget" name="NewTaskPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="newTaskOtsuButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Otsu Threshold</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newTaskImageProcessButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Image Preprocessing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newTaskIndexedOrgansButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Indexed Organ Statistics</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newTaskTotalSegmentatorButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Segmentator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
